docs/editors/debug-adapter-protocol.md

```diff
@@ -57,6 +58,10 @@ Apart from using code lenses, users can start a debug session by executing the
 identically named classes in different modules. A uri will be returned that can
 be used by the DAP client.
 
+`envFile` is an optional parameter, which allows you to specify a path to a
+`.env` file with additional environment variables. It supports single line as
+well as multi-line quoted values (without value substitution).
+
```

Comment:
Add clarification in docs what takes precedense in case of parameter defined in `envFile` and `env` parameter

Reply:
done in 21158cd
I've also updated one of the tests to reflect the precedence.